bundles/org.openhab.binding.meteoalerte/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,211 @@
+					<option value="HAUTES-PYRENEES">Hautes Pyrénées</option>
+					<option value="HAUTS-DE-SEINE">Hauts de Seine</option>
+					<option value="HERAULT">Hérault</option>
+					<option value="ILLE-ET-VILAINE">Ille et Vilaine</option>
+					<option value="INDRE">Indre</option>
+					<option value="INDRE-ET-LOIRE">Indre et Loire</option>
+					<option value="ISERE">Isère</option>
+					<option value="JURA">Jura</option>
+					<option value="LANDES">Landes</option>
+					<option value="LOIR-ET-CHER">Loir et Cher</option>
+					<option value="LOIRE">Loire</option>
+					<option value="LOIRE-ATLANTIQUE">Loire Atlantique</option>
+					<option value="LOIRET">Loiret</option>
+					<option value="LOT">Lot</option>
+					<option value="LOT-ET-GARONNE">Lot et Garonne</option>
+					<option value="LOZERE">Lozère</option>
+					<option value="MAINE-ET-LOIRE">Maine et Loire</option>
+					<option value="MANCHE">Manche</option>
+					<option value="MARNE">Marne</option>
+					<option value="MAYENNE">Mayenne</option>
+					<option value="MEURTHE-ET-MOSELLE">Meurthe et Moselle</option>
+					<option value="MEUSE">Meuse</option>
+					<option value="MORBIHAN">Morbihan</option>
+					<option value="MOSELLE">Moselle</option>
+					<option value="NIEVRE">Nièvre</option>
+					<option value="NORD">Nord</option>
+					<option value="OISE">Oise</option>
+					<option value="ORNE">Orne</option>
+					<option value="PARIS">Paris</option>
+					<option value="PAS-DE-CALAIS">Pas de Calais</option>
+					<option value="PUY-DE-DOME">Puy de Dôme</option>
+					<option value="PYRENEES-ATLANTIQUES">Pyrénées Atlantiques</option>
+					<option value="PYRENEES-ORIENTALES">Pyrénées Orientales</option>
+					<option value="RHONE">Rhône</option>
+					<option value="SAONE-ET-LOIRE">Saône et Loire</option>
+					<option value="SARTHE">Sarthe</option>
+					<option value="SAVOIE">Savoie</option>
+					<option value="SEINE-ET-MARNE">Seine et Marne</option>
+					<option value="SEINE-MARITIME">Seine Maritime</option>
+					<option value="SEINE-SAINT-DENIS">Seine Saint Denis</option>
+					<option value="SOMME">Somme</option>
+					<option value="TARN">Tarn</option>
+					<option value="TARN-ET-GARONNE">Tarn et Garonne</option>
+					<option value="TERRITOIRE DE BELFORT">Territoire de Belfort</option>
+					<option value="VAL-D'OISE">Val D'Oise</option>
+					<option value="VAL-DE-MARNE">Val de Marne</option>
+					<option value="VAR">Var</option>
+					<option value="VAUCLUSE">Vaucluse</option>
+					<option value="VENDEE">Vendée</option>
+					<option value="VIENNE">Vienne</option>
+					<option value="VOSGES">Vosges</option>
+					<option value="YONNE">Yonne</option>
+					<option value="YVELINES">Yvelines</option>
+				</options>
+				<limitToOptions>true</limitToOptions>
+			</parameter>
+			<parameter name="refresh" type="integer" min="1" required="true" unit="h">
+				<label>Fréquence de rafraichissement</label>
+				<description>Période d'actualisation des données.</description>
```

Comment:
Keep the info in the description that this setting unit is hour (because Paper UI does not display the unit). Without the info in the description, the user will have no idea what value to set.